src/components/layers-panel/layer-manager/LayerManager.ts

```diff
@@ -66,10 +77,21 @@ export class LayerManager extends i18nMixin(LitElement) {
                 .isOpen=${this.isLayerOpenId === layer.id}
                 @clickRemove="${this.removeLayer}"
                 @clickToggle="${this.toggleAccordionItem}"
+                @clickInfo="${this.toggleInfoLayer}"
                 @changeOpacity="${this.changeOpacityLayer}"
-              ></lux-layer-manager-element>
+              >
+              </lux-layer-manager-element>
             </li>`
         )}
+        ${this.backgroundLayer &&
+        html`
+          <lux-layer-manager-background-element
```

Comment:
it's not working on this branch, but maybe requires a merge of the background PR

Reply:
indeed, requires background selector PR to work